editoast/src/views/rolling_stocks.rs

```diff
@@ -30,18 +29,17 @@ mod tests {
     use actix_web::web::Data;
     use diesel::r2d2::{ConnectionManager, Pool};
     use diesel::PgConnection;
-    use std::fs;
 
     #[actix_test]
     async fn get_rolling_stock() {
         let app = create_test_service().await;
         let manager = ConnectionManager::<PgConnection>::new(PostgresConfig::default().url());
         let db_pool = Data::new(Pool::builder().max_size(2).build(manager).unwrap());
 
-        let path = "./src/tests/example_rolling_stock.json";
-        let data = fs::read_to_string(path).expect("Unable to read file");
-        let rolling_stock_form: RollingStockForm =
-            serde_json::from_str(&data).expect("Unable to parse");
+        let mut rolling_stock_form: RollingStockForm =
+            serde_json::from_str(&include_str!("../tests/example_rolling_stock.json"))
```

Comment:
In nightly clippy trigger a warning for an unused ref. It's not a big deal since the CI runs in stable.

```suggestion
            serde_json::from_str(include_str!("../tests/example_rolling_stock.json"))
```